libcnb-test/src/lib.rs

```diff
@@ -19,6 +19,7 @@ mod test_runner;
 mod util;
 
 pub use crate::container_context::*;
+pub use crate::log::*;
```

Comment:
Strictly speaking, do we need this? From the description, `crate::log::LogOutput` was exposed already? Is the idea there that we might as well expose it here?

Reply:
It fixed the generated rustdocs - I don't know if there is a better way to fix it, or if if fact it was already supposed to work the way it was and we were instead running into a rustdocs bug?

Reply:
Ah! Now I see what you mean - we expose values of this type, but since the type itself is not public, it will not be in the generated rustdocs. Gotcha. Yeah - then we have to do this. Thanks for clarifying! 👍🏻